src/snowflake/snowpark/modin/pandas/indexing.py

```diff
@@ -1181,7 +1189,15 @@ def __getitem__(
 
         if isinstance(result, Series):
             result._parent = self.df
-            result._parent_axis = 0
+            # We need to determine which axis this Series was extracted from. We can do so
```

Comment:
This looks like redundant code with above, would it make sense to put this in a separate helper function?